### PR TITLE
template all the things

### PR DESCRIPTION
**Pull request by gidsg**
Shifts more fields to the sheet and assets to the grid.
Aim is to minimise configuration outside of the sheet and config file.

CC @duarte @mr-mr @akash1810

### CODE REVIEW

File: src/js/lib/sheettodom.js [hunk 1]
```diff
@@ -21,5 +21,10 @@ export default function sheettodom(sheetID, sheetName, el, callback) {
 
         }
     });
+}
 
+function setAttribute(dom, selector, attribute, value){
```
Comment (akash1810) [inline src/js/lib/sheettodom.js]: is `dom` a Node rather than document?

**Comment by akash1810** [comment]
Grid crops have different size assets to them - do we need to resize the images in css in case a super large one is chosen?

**Comment by gidsg** [comment]
I have just been doing a freeform crop to the exact required size.

**Comment by akash1810** [comment]
👍

**Pull request merged**